### DISCUSSION TITLE
0.4 Milestone Timeline

### DISCUSSION THREAD

**Discussion by asheeshgarg** [microsoft/autogen#4460] [General]
Do we have a timeline for the first release of the 0.4 ?

**Comment by ekzhu** [comment]
Track our progress here: https://github.com/microsoft/autogen/issues?q=is%3Aissue%20state%3Aopen%20milestone%3A0.4.0

You are welcome to join our office hour to discuss. https://github.com/microsoft/autogen/discussions/4059

We will have a discord up soon.
